test/space/ntuples_test.py

```diff
@@ -300,15 +362,6 @@ def idiv_aliased(x):
     _test_unary_operator(fn, lambda x: x / x)
 
 
-def test_norm(fn):
```

Comment:
That one only got reshuffled. Regarding weighting, I'd say tests for the `FnWeighting` classes should cover this, no need to double up weighting tests here (beyond init and equality tests, of course). The tests for the weighting classes are desperately needed, though.